src/middlewared/middlewared/plugins/mdns.py

```diff
@@ -81,6 +151,9 @@ def __init__(self, **kwargs):
         self.logger = kwargs.get('logger')
         self.timeout = kwargs.get('timeout', 30)
 
+        self.mdnsd = mDNSDaemonThread()
```

Comment:
Why is it called `mDNSDaemonThread`? Name like `mDNSStartupWaiter` will have more sense.

Reply:
Because it was formerly a thread. Code was rearranged a little bit.